requirements/tests.txt

```diff
@@ -18,3 +18,4 @@ matplotlib
 cython
 flaky
 coverage <5.0
+rtree
```

Comment:
I don't understand why you need to add `rtree` here, when it's already part of `conda.txt`:

https://github.com/spyder-ide/spyder/blob/master/requirements/conda.txt#L28

Reply:
Oh I though it was not there. I will revert then this change :+1: